libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/summary/SummaryBottomSheet.java

```diff
@@ -150,4 +156,18 @@ private void clearViews() {
     timeRemainingText.setText(EMPTY_STRING);
     distanceRemainingText.setText(EMPTY_STRING);
   }
+
+  public void setLocale(Locale locale) {
+    this.locale = locale;
+  }
+
+  public void setUnitType(@NavigationUnitType.UnitType int unitType) {
+    this.unitType = unitType;
+  }
+
+  private void verifyLocale() {
```

Comment:
It looks like this logic is duplicated for `InstructionView`, `SummaryBottomSheet` and `MapboxNavigationNotification`, maybe extract it into `LocaleUtils#verifyLocale(Context, Locale)`?  

It can take a given `Locale` and if not null, just return the passed `Locale`.  If it is null, you can return `LocaleUtils.getDeviceLocale(getContext())`

This can also replace https://github.com/mapbox/mapbox-navigation-android/pull/691/files#diff-27393dc3fcfc5c75694053a5050c0d40R130